flink-runtime/src/test/java/org/apache/flink/runtime/io/network/buffer/BufferBuilderTestUtils.java

```diff
@@ -67,18 +67,22 @@ public static Buffer buildSingleBuffer(BufferConsumer bufferConsumer) {
 	}
 
 	public static BufferConsumer createFilledFinishedBufferConsumer(int dataSize) {
-		return createFilledBufferConsumer(dataSize, dataSize, true);
+		return createFilledBufferConsumer(dataSize, dataSize, true, true);
```

Comment:
Why changed the previous tests to use the `isShareable` property by default? 
The shareable property is not the general scenarios and is only used for broadcast with compression. So it is enough to only cover the compression case for verifying. The most common cases should be non-shareable.